cmd/di.go

```diff
@@ -51,6 +51,7 @@ import (
 	"github.com/mysteriumnetwork/node/money"
 	"github.com/mysteriumnetwork/node/nat"
 	"github.com/mysteriumnetwork/node/nat/mapping"
+	"github.com/mysteriumnetwork/node/nat/natevents"
```

Comment:
This package is already in the `nat` directory, we probably should not repeat the name in the subpackage: 
```suggestion
	"github.com/mysteriumnetwork/node/nat/event"
```

And go style guide recommends not to use the plural for the package names: 
> In go, package names are not plural.